provider/resources.go

```diff
@@ -147,411 +139,99 @@ func Provider() tfbridge.ProviderInfo {
-					"account_id": makeStringID(),
-				},
-			},
-			"newrelic_cloud_aws_link_account": {
-				Tok: makeResource(cloudMod, "AwsLinkAccount"),
-				Fields: map[string]*tfbridge.SchemaInfo{
-					"account_id": makeStringID(),
-				},
-			},
-			"newrelic_cloud_azure_link_account": {
-				Tok: makeResource(cloudMod, "AzureLinkAccount"),
-				Fields: map[string]*tfbridge.SchemaInfo{
-					"account_id": makeStringID(),
-				},
-			},
-			"newrelic_cloud_gcp_link_account": {
-				Tok: makeResource(cloudMod, "GcpLinkAccount"),
-				Fields: map[string]*tfbridge.SchemaInfo{
-					"account_id": makeStringID(),
-				},
-			},
-			"newrelic_cloud_aws_govcloud_integrations": {
-				Tok: makeResource(cloudMod, "AwsGovcloudIntegrations"),
-				Fields: map[string]*tfbridge.SchemaInfo{
-					"account_id":        makeStringID(),
-					"linked_account_id": makeStringID(),
-				},
-			},
-			"newrelic_cloud_aws_integrations": {
-				Tok: makeResource(cloudMod, "AwsIntegrations"),
-				Fields: map[string]*tfbridge.SchemaInfo{
-					"account_id":        makeStringID(),
-					"linked_account_id": makeStringID(),
-				},
-			},
-			"newrelic_cloud_azure_integrations": {
-				Tok: makeResource(cloudMod, "AzureIntegrations"),
-				Fields: map[string]*tfbridge.SchemaInfo{
-					"account_id":        makeStringID(),
-					"linked_account_id": makeStringID(),
-				},
-			},
-			"newrelic_cloud_gcp_integrations": {
-				Tok: makeResource(cloudMod, "GcpIntegrations"),
-				Fields: map[string]*tfbridge.SchemaInfo{
-					"account_id":        makeStringID(),
-					"linked_account_id": makeStringID(),
-				},
-			},
+			"newrelic_cloud_aws_govcloud_link_account": {Tok: makeResource(cloudMod, "AwsGovcloudLinkAccount")},
+			"newrelic_cloud_aws_link_account":          {Tok: makeResource(cloudMod, "AwsLinkAccount")},
+			"newrelic_cloud_azure_link_account":        {Tok: makeResource(cloudMod, "AzureLinkAccount")},
+			"newrelic_cloud_gcp_link_account":          {Tok: makeResource(cloudMod, "GcpLinkAccount")},
+			"newrelic_cloud_aws_govcloud_integrations": {Tok: makeResource(cloudMod, "AwsGovcloudIntegrations")},
+			"newrelic_cloud_aws_integrations":          {Tok: makeResource(cloudMod, "AwsIntegrations")},
+			"newrelic_cloud_azure_integrations":        {Tok: makeResource(cloudMod, "AzureIntegrations")},
+			"newrelic_cloud_gcp_integrations":          {Tok: makeResource(cloudMod, "GcpIntegrations")},
 
-			"newrelic_synthetics_alert_condition": {
-				Tok: makeResource(syntheticsMod, "AlertCondition"),
-				Fields: map[string]*tfbridge.SchemaInfo{
-					"policy_id": makeStringID(),
-				},
-			},
-			"newrelic_synthetics_monitor": {
-				Tok: makeResource(syntheticsMod, "Monitor"),
-				Fields: map[string]*tfbridge.SchemaInfo{
-					"account_id": makeStringID(),
-				},
-			},
-			"newrelic_monitor_downtime": {
-				Tok: makeResource(syntheticsMod, "MonitorDowntime"),
-				Fields: map[string]*tfbridge.SchemaInfo{
-					"account_id": makeStringID(),
-				},
-			},
```

Comment:
We might have lost `MonitorDowntime` here, which I think belongs in the `synthetics` package

Reply:
We have. I wasn't sure why it had moved. Why do you think that `MonitorDowntime` (`newrelic_monitor_downtime`) belongs in the `synthetics` (`newrelic_synthetics_*`) package?

Reply:
We definitely don't have to move it of course, but my impression is that it is linked to the synthetic features of NewRelic and that TF have incorrectly put it at the top-level (see screenshots). But if we'd rather keep it where it is that's fine too!

![image](https://github.com/pulumi/pulumi-newrelic/assets/163793/e8d8bfc6-e149-44c0-8f7f-970f3de9edd5)
![image](https://github.com/pulumi/pulumi-newrelic/assets/163793/549b0ec7-82d1-40dc-80c7-6d1adc2ed649)
![image](https://github.com/pulumi/pulumi-newrelic/assets/163793/526a9ab3-b96f-4df3-aae1-600c02f6cd4c)
![image](https://github.com/pulumi/pulumi-newrelic/assets/163793/7f18fb10-260e-4efd-b272-c3713b57afca)
![image](https://github.com/pulumi/pulumi-newrelic/assets/163793/6ce548de-cd11-45b0-903e-0f923fd385a3)